build/Packages.props

```diff
@@ -1,89 +1,90 @@
-    <PackageReference Update="Microsoft.Extensions.DependencyInjection" Version="2.1.1" />
-    <PackageReference Update="Microsoft.Extensions.DependencyModel" Version="2.1.0" />
-    <PackageReference Update="Microsoft.Extensions.FileProviders.Physical" Version="2.1.1" />
-    <PackageReference Update="Microsoft.Extensions.FileSystemGlobbing" Version="2.1.1" />
-    <PackageReference Update="Microsoft.Extensions.Logging" Version="2.1.1" />
-    <PackageReference Update="Microsoft.Extensions.Logging.Console" Version="2.1.1" />
-    <PackageReference Update="Microsoft.Extensions.Options" Version="2.1.1" />
-    <PackageReference Update="Microsoft.Extensions.Options.ConfigurationExtensions" Version="2.1.1" />
-
-    <PackageReference Update="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0-preview.1" />
-    <PackageReference Update="Microsoft.NET.Test.Sdk" Version="15.7.2" />
-    <PackageReference Update="Microsoft.TestPlatform.TranslationLayer" Version="15.7.2" />
-    <PackageReference Update="Microsoft.VisualStudio.CodingConventions" Version="1.1.20180503.2" />
-    <PackageReference Update="Microsoft.VisualStudio.Setup.Configuration.Interop" Version="1.14.114" />
-    <PackageReference Update="Microsoft.VisualStudio.SDK.EmbedInteropTypes" Version="15.0.12" />
-
-    <PackageReference Update="Newtonsoft.Json" Version="9.0.1" />
-
-    <PackageReference Update="Nuget.Packaging" Version="$(NuGetPackageVersion)" />
-    <PackageReference Update="Nuget.Packaging.Core" Version="$(NuGetPackageVersion)" />
-    <PackageReference Update="Nuget.ProjectModel" Version="$(NuGetPackageVersion)" />
-    <PackageReference Update="Nuget.Versioning" Version="$(NuGetPackageVersion)" />
-
-    <PackageReference Update="OmniSharp.Extensions.LanguageServer" Version="0.13.1" />
-
-    <PackageReference Update="SQLitePCLRaw.bundle_green" Version="1.1.2" />
-    <PackageReference Update="System.Collections.Immutable" Version="1.4.0" />
-    <PackageReference Update="System.ComponentModel.Composition" Version="4.5.0" />
-    <PackageReference Update="System.Composition" Version="1.0.31" />
-    <PackageReference Update="System.Reflection.DispatchProxy" Version="4.5.1" />
-    <PackageReference Update="System.Reflection.Metadata" Version="1.4.2" />
-    <PackageReference Update="System.Threading.Tasks.Dataflow" Version="4.9.0" />
-    <PackageReference Update="System.ValueTuple" Version="4.4.0" />
-
-    <PackageReference Update="System.Reactive" Version="4.1.2" />
-
-    <PackageReference Update="System.Reflection.DispatchProxy" Version="4.5.1" />
-
-    <PackageReference Update="xunit.runner.visualstudio" Version="$(XunitPackageVersion)" />
-    <PackageReference Update="xunit" Version="$(XunitPackageVersion)" />
-  </ItemGroup>
-  <ItemGroup>
-    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0-beta2-18618-05" PrivateAssets="All" />
-  </ItemGroup>
+<Project ToolsVersion="14.0"
```

Comment:
could you please reset the changes to this file so that only the relevant lines are updated? we often look here at the blame to identify when a specific dependency was introduced - and this will break it